docs/source/concepts/identity-and-package-management.rst

```diff
@@ -0,0 +1,170 @@
+The remainder of the document will presents both:
+
+#. the minimal behavioral guarantees for identity and package services across all ledger implementations, and
+
+#. guidelines to understand how the :ref:`ledger's topology <daml-ledger-topologies>` influences the unspecified part of the behavior.
+
+Identity Management
+*******************
+
+A DAML Ledger may freely define its own format of party and :ref:`participant node <participant-node-def>` identifiers, with some minor constraints on the identifiers' serialized form.
+For example, a ledger may use human-readable strings as identifiers, such as "Alice" or "Alice's Bank".
+A different ledger might use public keys as identifiers, or the keys' fingerprints.
+The applications should thus not rely on the format of the identifier -- even a software upgrade of a DAML Ledger may introduce a new format.
+
+By definition, identifiers identify parties, and are thus unique for a ledger.
+They do not, however, have to be unique across different ledgers.
+That is, two parties with identical identifiers in two different ledgers are not the same party.
+Moreover, a real-world entity can have multiple identifiers (and thus parties) within the same ledger.
+
+Since the identifiers might be difficult to interpret and manage for humans, the ledger may also accompany each identifier with a user-friendly **display name**.
+Unlike the identifier, the display name is not guaranteed to be unique.
+Furthermore, a display name is in general not guaranteed to have any link to real world identities.
+For example, a party with a display name "Attorney of Nigerian Prince" might well be controlled by a real-world entity without a bar exam.
+However, particular ledger deployments might make stronger guarantees about this link.
+Finally, the association of identifiers to display names may change over time.
+For example, a party might change its display name from "Bruce" to "Caitlyn" -- as long as the identifier remains the same, so does the party.
+
+.. _provisioning-ledger-identifiers:
+
+Provisioning Identifiers
+========================
+
+The set of parties of any DAML Ledger is dynamic: new parties may always be added to the system.
+The first step in adding a new party to the ledger is to provision a new identifier for the party.
+The Ledger API provides an ``AllocateParty`` method for this purpose.
+The method, if successful, returns an new party identifier.
+The ``AllocateParty`` call can take the desired identifier and displayed name as optional parameters, but these are merely hints and the ledger implementation may completely ignore them.
+
+If the call returns a new identifier, the :ref:`participant node <participant-node-def>` serving this call is ready to host the party with this identifier.
+The returned identifier is guaranteed to be **unique** in the ledger; namely, no other call of the ``AllocateParty`` method at this or any other ledger participant may return the same identifier.
```

Comment:
Again, this doesn't make sense to me in a Canton world where ledgers can merge and split.

Reply:
I'm not sure what issue you're raising here: is the statement too weak, or too strong? 

I don't see any stronger statements we could make: we can only guarantee that the returned identifier is unique within "the ledger". I can easily have two Sandbox instances who return the same identifiers when ``AllocateParty`` is called. 

I don't think it's too weak either; I believe it should hold for all ledgers with central operators, and also for Canton. Even when Canton is viewed as a single virtual global ledger, ``AllocateParty`` will never return the same identifier, even when initiated at two participants who are cut off from each other (at least with the current Canton identity management system).

Reply:
It's too strong: in Canton we can have two participants that share the same private key, and thus allocate the same party identifiers. I'll rephrase to something weaker.